packages/backend/src/api/routes/health.ts

```diff
@@ -0,0 +1,37 @@
+import type { ApiRoutes } from '@vse-bude/shared';
+import { Router } from 'express';
+import { PrismaClient } from '@prisma/client';
+import { initServices } from '@services';
+import { initRepositories } from '@repositories';
+
+function initHealthRoutes(path: ApiRoutes) {
+  const router = Router();
+  let status: number;
+
+  const prisma = new PrismaClient({
```

Comment:
Please, don't initialize Prisma and redis server in initHealthRoutes. Create a repository and a service for db healthceck. Pass redisService and dbHealthService in initRoutes in initHealthRoutes as it done with other routes. Use this services here.